cli/cfncluster/cfncluster.py

```diff
@@ -403,6 +432,62 @@ def instances(args):
     for instance in instances:
         print('%s         %s' % (instance[0],instance[1]))
 
+def get_head_user(parameters, template):
+    try:
+        mappings = json.loads(
+                template.get("GetTemplateResponse") \
+                .get("GetTemplateResult") \
+                .get("TemplateBody")) \
+                .get("Mappings") \
+                .get("OSFeatures")
+        base_os =[i.value for i in parameters if i.key == "BaseOS"][0]
+        return mappings.get(base_os).get("User")
+    except json.decoder.JSONDecodeError or AttributeError as e:
+        logger.error("Could not get username.", e)
+        sys.exit(1)
+
+def command(args, extra_args):
+    stack = ('cfncluster-' + args.cluster_name)
+    config = cfnconfig.CfnClusterConfig(args)
+    if args.command in config.aliases:
+        config_command = config.aliases[args.command]
+    else:
+        logger.info("Command %s not valid" % args.command)
+        sys.exit(1)
+
+    cfnconn = boto.cloudformation.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
+                                                    aws_secret_access_key=config.aws_secret_access_key)
+    try:
+        status = cfnconn.describe_stacks(stack)[0].stack_status
+        invalid_status = ['DELETE_COMPLETE', 'DELETE_IN_PROGRESS']
+        if status in invalid_status:
+            logger.info("Stack status: %s. Needs to be in %s" % (status, ' or '.join(invalid_status)))
+            sys.exit(1)
+        ip = get_master_server_ip(stack, config)
+        stack_result = cfnconn.describe_stacks(stack)[0]
+        template = cfnconn.get_template(stack)
+        username = get_head_user(stack_result.parameters, template)
+
+        # build command
+        cmd = config_command.format(CFN_USER=username, MASTER_IP=ip, ARGS=' '.join(str(e) for e in extra_args))
```

Comment:
Is this going to work if `extra_args` contains spaces, e.g. `-i "/path with spaces/to/pemfile"` ? 
I may be wrong but I'd expect the join would correctly manage the path above as a single param but final result for `ARGS` and so the command would be wrong.
It would be great to have a unit test for it.